sledgehammer/start-up.sh

```diff
@@ -149,6 +149,11 @@ echo "# Sledgehammer added to log to the admin node" >> /etc/rsyslog.conf
 echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
+# Sometimes at this point network is not up yet, wait for it
```

Comment:
Can you put something like:
 echo "Waiting for admin server to be reachable; will wait up to 60 seconds..."